source/puppet/4.1/reference/config_ssl_external_ca.markdown

```diff
@@ -1,29 +1,27 @@
 ---
 layout: default
 title: "SSL Configuration: External CA Support"
-canonical: "/puppet/latest/reference/config_ssl_external_ca.html"
+canonical: "/puppet/4.1/reference/config_ssl_external_ca.html"
```

Comment:
Canonical needs to stay set to latest; this helps prevent google from sending people to outdated pages.